src/net/db.rs

```diff
@@ -85,6 +85,7 @@ impl FromColumn<PeerAddress> for PeerAddress {
 #[derive(PartialEq, Clone)]
 pub struct LocalPeer {
     pub network_id: u32,
+    pub parent_network_id: u32,
```

Comment:
Is `parent_network_id` the burnchain network ID?

Reply:
Yes